content/en/stats.md

```diff
@@ -4,6 +4,7 @@ linkTitle: Statistics
 slug: stats
 top_graphic: 3
 excerpt: Let's Encrypt certificate statistics.
+lastmod: 2019-22-08
```

Comment:
@tdelmas Since the format is `YYYY-MM-DD`, should this `lastmod` string be `2019-08-22`, not `2019-22-08`?

Reply:
Yes! Thank you!

Reply:
Fixed in https://github.com/letsencrypt/website/commit/896f89d6188244d3be29af6c9904d9481559334c

Reply:
Great!